coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -32,14 +34,20 @@ def __init__(self, documentation, docstyle_definition,
         :param marker:
             The three-element tuple with marker strings, that identified this
             documentation comment.
-        :param range:
-            The position range of type ``TextRange``.
+        :param position:
+            The starting ``TextPosition`` of the documentation.
         """
         self.documentation = documentation
         self.docstyle_definition = docstyle_definition
         self.indent = indent
         self.marker = marker
-        self.range = range
+        self.position = position
+        self.assemble = None if position is None else self.assemble()
```

Comment:
I guess you're asking for `self.position` yep it's required. Is currently being used in tests and later in `DocStyleBear`. 
For `self.assemble` it's being used to construct that later `self.range`